changelog/3828.bugfix.rst

```diff
@@ -0,0 +1 @@
+- Fixed the issue of broken link. Details of the issue can be found at #3744. (`#3744 https://github.com/sunpy/sunpy/pull/3828`__)
```

Comment:
```suggestion
Fixed broken "edit on github" links in the documentation.
```
You don't have to link to the issue explicitly, and I've suggested a more descriptive sentence.